css/style.css

```diff
@@ -60,4 +60,92 @@
 .button--danger {
 	color: #ffffff;
 	background-color: #ce4141;
-}
\ No newline at end of file
+}
+/*Comodities */
+html {
+  font-family: "Roboto", sans-serif;
+  margin: 20px;
+}
+@media (max-width: 768px) {
+  img {
+    width: 100%;
+    height: 100%;
+    max-height: 233px;
+    max-width: 367px;
+  }
+  .carrouselContainer {
```

Comment:
This name doesn't match with BEM naming convention. Please check the documentation.